docs/postgres.rst

```diff
@@ -6,7 +6,19 @@ Postgres version 9.4 or later is known to work.
 Set up database
 ===============
 
-The PostgreSQL database used *must* have the correct encoding set, otherwise
+Assuming your PostgreSQL database user is called ``postgres``, create a user
```

Comment:
I am not sure if this is the most elegant solution to create the user, I have no system at the moment to try out a shorter solution. 

feel free to edit this section if there is a shorter solution.